### PR TITLE
Adding new page to Annotations for the feature show and hide annotations

### PR DESCRIPTION
**Pull request by DeepalakshmiSivasooriyan**
Title : UG content preparation for Support to show and hide annotations in .NETMAUI SfPdfViewer
Task Link : https://dev.azure.com/EssentialStudio/Mobile%20and%20Desktop/_workitems/edit/890339

### CODE REVIEW

**Comment by kannans24** [comment]
rebuild

File: MAUI/PDF-Viewer/Show-Hide.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+description: Learn here about showing and hiding annotations in a document in Syncfusion .NET MAUI PDF Viewer (SfPdfViewer) control and more.
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+keywords: .net maui pdf viewer, .net maui view pdf, pdf viewer in .net maui, .net maui open pdf, maui pdf viewer, maui pdf view
+---
+# Show and Hide annotations in .NET MAUI PDF Viewer (SfPdfViewer)
+ 
+You can manage the visibility of annotations using the `Hidden` property. It helps you to hide annotations containing confidential data when sharing documents externally or presenting documents, ensuring data privacy and security. Also, it makes it easier to read the documents which are cluttered with more annotations. Like other annotation properties, you can undo and redo the hiding or showing actions. Additionally, when an annotation is locked, it cannot be hidden. Annotations that are hidden will remain hidden during import, export, printing and saving
+##Hide annotations
```
Comment (Deepak1799) [inline MAUI/PDF-Viewer/Show-Hide.md:13]: The header is not properly defined

File: MAUI/PDF-Viewer/Show-Hide.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+title: Show and Hide annotations in .NET MAUI PDF Viewer | Syncfusion
+description: Learn here about showing and hiding annotations in a document in Syncfusion .NET MAUI PDF Viewer (SfPdfViewer) control and more.
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+keywords: .net maui pdf viewer, .net maui view pdf, pdf viewer in .net maui, .net maui open pdf, maui pdf viewer, maui pdf view
+---
+# Show and Hide annotations in .NET MAUI PDF Viewer (SfPdfViewer)
+ 
+You can manage the visibility of annotations using the `Hidden` property. It helps you to hide annotations containing confidential data when sharing documents externally or presenting documents, ensuring data privacy and security. Also, it makes it easier to read the documents which are cluttered with more annotations. Like other annotation properties, you can undo and redo the hiding or showing actions. Additionally, when an annotation is locked, it cannot be hidden. Annotations that are hidden will remain hidden during import, export, printing and saving
```
Comment (Deepak1799) [inline MAUI/PDF-Viewer/Show-Hide.md:12]: At the end of the paragraph full stop is missing

**Review by Deepak1799 — CHANGES_REQUESTED** [review]
Requested changes

File: MAUI/PDF-Viewer/Show-Hide.md (hunk 1 above)
Comment (DeepalakshmiSivasooriyan) [reply MAUI/PDF-Viewer/Show-Hide.md]: I have changed the definition of header sir

File: MAUI/PDF-Viewer/Show-Hide.md (hunk 2 above)
Comment (DeepalakshmiSivasooriyan) [reply MAUI/PDF-Viewer/Show-Hide.md]: I have corrected it now sir

File: MAUI/PDF-Viewer/Show-Hide.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            annotation.Hidden = true; 
+        } 
+  }            
+}
+{% endhighlight %}
+{% endtabs %}
+
+* Similarly, to show the annotation, set the `Hidden` property value to `false`. 
+
+## Note:
```
Comment (Deepak1799) [inline MAUI/PDF-Viewer/Show-Hide.md:81]: Note should be added as tag, not as header. Please cross check existing md files

**Review by Deepak1799 — CHANGES_REQUESTED** [review]
Requested changes

**Review by Deepak1799 — APPROVED** [review]
Approved the changes

**Pull request merged**